### PR TITLE
[usecase-wrt] Add notification checking for download progress

### PR DESCRIPTION
**Pull request by haoxli**
Add check point for download progress notification, such as
finished notification, and notification for open downloaded file.

Impacted tests(approved): new 0, update 1, delete 0
Unit test platform: Crosswalk for Android 20.49.527.0
Unit test result summary: pass 1, fail 0, block 0

BUG=https://crosswalk-project.org/jira/browse/CTS-904

### CODE REVIEW

**Comment by haoxli** [comment]
@Honry @yhe39

File: usecase/usecase-wrt-android-tests/samples/PdfDownload/res/index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<!--
+Copyright (c) 2015 Intel Corporation.
```
Comment (yhe39) [inline usecase/usecase-wrt-android-tests/samples/PdfDownload/res/index.html]: s/2015/2016/g

File: usecase/usecase-wrt-android-tests/samples/PdfDownload/res/index.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ARE DISCLAIMED. IN NO EVENT SHALL INTEL CORPORATION BE LIABLE FOR ANY DIRECT,
+INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+Authors:
+        Wang, Hongjuan <hongjuanx.wang@intel.com>
```
Comment (yhe39) [inline usecase/usecase-wrt-android-tests/samples/PdfDownload/res/index.html]: add you in the Authors

File: usecase/usecase-wrt-android-tests/samples/PdfDownload/res/index.html (hunk 1 above)
Comment (haoxli) [inline usecase/usecase-wrt-android-tests/samples/PdfDownload/res/index.html]: For test update, no need update this.

**Comment by haoxli** [comment]
@yhe39 updated

**Comment by yhe39** [comment]
LGTM

**Pull request merged**